internal/service/project/resource_project.go

```diff
@@ -245,6 +254,16 @@ func (r *projectRS) Schema(ctx context.Context, req resource.SchemaRequest, resp
 					},
 				},
 			},
+			"tags": schema.MapAttribute{
+				ElementType: types.StringType,
+				Optional:    true,
+				Computed:    true,
```

Comment:
As discussed offline, not sure if this should be Computed too, we can discuss more in the meeting if needed

Reply:
Agree that `Computed` does not seem accurate here. I would assume users will handle all configuration through terraform, and if there are exceptions then `ignore_changes` lifecycle can be used to ignore the `tags` attribute or specific tag keys. This forces the config to be explicit of what values are managed externally from terraform.
For reference [aws tags schema](https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/tags/tags.go#L11), as well as our existing [tags schema](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/5a7d85cbe9695abc89392a1c7d72d76b0b272d47/internal/service/advancedcluster/model_advanced_cluster.go#L38).

Reply:
Great point and references.
I'll do a little investigation 🔍

Reply:
Yes, no point in using `Computed`.
But I had to add an extra flag to have a difference between MapNull and MapEmpty.
`func NewTFTags(tags []admin.ResourceTag, useNullForEmpty bool) types.Map {`
https://github.com/mongodb/terraform-provider-mongodbatlas/blob/CLOUDP-242261_project_tags/internal/service/project/model_project.go#L210